releasenotes/notes/fix-dont-log-remoteconfig-stacktrace-ab7d1db057233454.yaml

```
@@ -0,0 +1,3 @@
+fixes:
+  - |
+    Fixed full stacktrace being sent to the log on remote config connection errors.
```

```suggestion
    Fix full stacktrace being sent to the log on remote config connection errors.
```
and is there an appropriate scope we can prefix? cc @majorgreys @mabdinur

---

Thanks, commited.